navbar.js

```diff
@@ -1,9 +1,166 @@
+    ],
+    live_version: 'placeholder',
+    source: 'https://github.com/Kelvin-Ben/Desktop-Version-Portfolio',
+    id: '3',
+  },
+  {
+    name: 'Uber Navigation',
+    longDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featuredimage: './images/SnapshootPortfolio4.png',
+    technologies: 'Uber',
+    role: 'Lead Developer',
+    year: '2018',
+    languages: [
+      'html', 'Ruby on rails', 'css', 'javascript',
+    ],
+    live_version: 'placeholder',
+    source: 'https://github.com/Kelvin-Ben/Desktop-Version-Portfolio',
+    id: '4',
+  },
+];
+
+const ret = (arr) => {
+  let lang = '';
+  arr.forEach((item) => {
+    lang += `<li>${item}</li>`;
+    return lang;
+  });
+  return lang;
+};
+
+const workcards = document.getElementById('Portfolio');
+
+projects.forEach((item) => {
+  workcards.innerHTML += `<article class="workcard" id=${item.id}>
+  <div>
+  <img src=${item.featuredimage} alt="" class="image">
+  </div>
+    <div class="block">
+      <h2 class="tonic1">${item.name}</h2>
+      <div class="technologies">
+      <div class="frame canopy">${item.technologies}</div>
+      <div class="frame counter"></div>
+      <div class="frame backend">${item.role}</div>
+      <div class="frame counter"></div>
+      <div class="frame year">${item.year}</div>
+    </div>
+    <p class="paragraph">${item.description}</p>
+      <div class="tags">
+        <ul class="tech">
+        ${ret(item.languages, item)}
+        </ul>
+      </div>
+      <button class="projects" id=${item.id} onclick="popupOpen(this.id)"><span class="see-projects">See Projects</span></button>
+    </div>
+  </article>
+  `;
+});
+
+const popup = document.getElementById('popup');
+
+const popupbutton = document.getElementsByClassName('projects');
+
+function popupClose() {
+  popup.style.display = 'none';
+}
+
+function popupOpen(id) {
+  popup.style.display = 'flex';
+  projects.forEach((item) => {
+    if (id === item.id) {
+      popup.innerHTML += `
```

Comment:
- [ ] When you click on `see project` button it supposes to dynamically render only the project for this specific work card not rendering all popup cards and change the display attribute to flex to make it appear. So please fix that.

![image](https://user-images.githubusercontent.com/68796232/189015571-1b602793-a399-45f3-acc8-8979702d371a.png)